lib/Dialect/Torch/Transforms/DecomposeComplexOpsEarly.cpp

```diff
@@ -92,12 +92,8 @@ class DecomposeAtenSplitTensorLikeOp
     auto listType = op.getType().template cast<Torch::ListType>();
     Location loc = op.getLoc();
 
-    auto viewOp = dyn_cast<AtenViewOp>(op.self().getDefiningOp());
-    if (!viewOp)
-      return failure();
-
     auto sizesListOp =
-        dyn_cast<PrimListConstructOp>(viewOp.size().getDefiningOp());
+        dyn_cast<PrimListConstructOp>(op.self().getDefiningOp());
```

Comment:
This is wrong. `op.self()` will never be a `PrimListConstructOp` because it is a tensor.